test/tools/router_check/test/route_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+# Router_check_tool binary path
+PATH_BIN="test/tools/router_check/router_check_tool"
+
+# Config json path
+PATH_CONFIG="test/tools/router_check/test/config"
+
+TESTS=("ContentType" "ClusterHeader" "HeaderMatchedRouting" "Redirect" "Redirect2" "Redirect3" "TestRoutes" "Weighted")
+
+# Testing expected matches
+for t in "${TESTS[@]}"
+do
+  TEST_OUTPUT=$("${PATH_BIN}" "${PATH_CONFIG}/${t}.json" "${PATH_CONFIG}/${t}.golden.json" "--details")
+done
+
+# Bad config file
+BAD_CONFIG_OUTPUT=$(("${PATH_BIN}" "${PATH_CONFIG}/Redirect.golden.json" "${PATH_CONFIG}/TestRoutes.json") 2>&1) ||
+  if [[ "${BAD_CONFIG_OUTPUT}" == *"config schema JSON load failed"* ]]; then
+    echo testing bad config output
```

Comment:
You should also tell the user what the expected output was and what was actually observed.